great_expectations/render/renderer/notebook_renderer.py

```diff
@@ -142,7 +142,7 @@ def add_expectation_cells_from_suite(self, expectations):
         if expectations_by_column["table_expectations"]:
             for exp in expectations_by_column["table_expectations"]:
                 kwargs_string = self._build_kwargs_string(exp)
-                code = "batch.{}({})".format(exp["expectation_type"], kwargs_string)
+                code = "batch.{}({}, include_config=False)".format(exp["expectation_type"], kwargs_string)
```

Comment:
This is an easy fix, however, I'd prefer it if users didn't have to see this additional `include_config=False` when working in notebooks. Ideally, the expectation output would be simplified via the jupyter_ux module. So when in a notebook and the user prints the result of an `expect....` method they'd see the typical json, but without the config.